zulipterminal/ui_tools/views.py

```diff
@@ -945,40 +944,26 @@ def __init__(self, controller: Any, color: str, desc: str,
 
 
 class MsgInfoView(PopUpView):
-    def __init__(self, controller: Any, msg: Message) -> None:
+    def __init__(self, controller: Any, msg: Message,
+                 popup_title: str) -> None:
         self.msg = msg
 
+        msg_info = [
+            ('', [('Date & Time', time.ctime(msg['timestamp'])[:-5]),
+                  ('Sender', msg['sender_full_name']),
+                  ('Sender\'s Email ID', msg['sender_email'])]),
+        ]
```

Comment:
I have grouped these into a category as per the discussion in the other comment.